.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: elixir
+
+elixir: '1.5'
+
+script:
+  # This is basically `mix test` with coverage enabled.
+  - mix coveralls.json
```

Comment:
Do you not want to do https://github.com/parroty/excoveralls#mix-coverallstravis-post-coverage-from-travis ?

Reply:
That would post to coveralls.io, not codecov.io. :-)